### PR TITLE
OSDOCS-11802 Information added to 4.14.z Known Issue

### PR DESCRIPTION
**Pull request by cbippley**
Version(s): 4.14

Issue: [OSDOCS-11802](https://issues.redhat.com/browse/OSDOCS-11802)

Link to [docs preview](https://80962--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-34-known-issue_release-notes)

QE review:
- [X] QE has approved this change.

Additional information: Needs Change Management.

### CODE REVIEW

**Comment by cbippley** [comment]
@sferich888 @xltian Can you ACK this PR for the change management process? Thank you!

**Comment by kalexand-rh** [comment]
LGTM

**Comment by mwerner2113** [comment]
LGTM

**Comment by xltian** [comment]
LGTM

**Comment by fbaudin** [comment]
LGTM!

**Comment by sferich888** [comment]
lgtm

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3631,7 +3631,7 @@ $ oc adm release info 4.14.34 --pullspecs
 [id="ocp-4-14-34-known-issue_{context}"]
 ==== Known issues
 
-* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SRI-OV VF fail to create a pod sandbox and will not function. (link:https://issues.redhat.com/browse/OCPBUGS-38091[*OCPBUGS-38091*])
+* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SR-IOV VF fail with the error message: `SRIOV-CNI failed to configure VF "failed t │ o set vf 0 vlan configuration - id 0, qos 0 and proto 802.1q: invalid argument"`. To resolve this issue, upgrade the SR-IOV Network Operator before upgrading {product-title}. This ensures that the fix to this issue is included in the SR-IOV Network Operator. (link:https://issues.redhat.com/browse/OCPBUGS-38091[*OCPBUGS-38091*])
```
Comment (SchSeba) [inline release_notes/ocp-4-14-release-notes.adoc:3634]: `failed t │ o set vf` -> `failed to set vf`

Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc:3634]: Fixed, thanks!

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -3531,7 +3531,8 @@ $ oc adm release info 4.14.34 --pullspecs
 [id="ocp-4-14-34-known-issue_{context}"]
 ==== Known issues
 
-* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SRI-OV VF fail to create a pod sandbox and will not function. (link:https://issues.redhat.com/browse/OCPBUGS-38091[*OCPBUGS-38091*])
+* On clusters with the SR-IOV Network Operator installed and configured, pods with a secondary interface of SR-IOV VF create a pod fail with the error message: `SRIOV-CNI failed to configure VF "failed t │
```
[the remark this answers is not in the document]
Comment (cbippley) [reply release_notes/ocp-4-14-release-notes.adoc]: Ran it by the SME and he said it looks good now.

**Pull request merged**